File: Keiken-frontend/src/app/entity/developper.ts
import {Project} from './project';

export class Developper {

  name: string
  githubUsername: string
  urlAvatar: string
  projects: Project[]

  constructor(name, githubUsername, urlAvatar, projects) {
    this.name = name
    this.urlAvatar = urlAvatar
    this.githubUsername = githubUsername
    this.projects = projects
  }

}

File: keiken-backend/src/main/java/com/keiken/community/entitys/Developper.java
package com.keiken.community.entitys;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

import javax.persistence.*;
import java.util.List;

@Entity
@Data @NoArgsConstructor @AllArgsConstructor @ToString
public class Developper {

    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String name;
    private String githubUsername;
    private String urlAvatar;
    @OneToMany(mappedBy="developper")
    private List<Project> projects;
}
